Bookish/Bookish.DataAccess/LibraryService.cs

```diff
@@ -25,11 +25,11 @@ public IEnumerable<Book> GetBooks()
         public Book GetCopies(string isbn)
```

Comment:
Instead of setting the database connection manually here, it can be injected. To do that:
1. Add an `IDbConnection` argument to the constructor for this class
2. In ConfigureServices in Startup.cs add
```
services.AddScoped<IDbConnection>(_ => new SqlConnection(defaultConnectionString));
```

🙂